docs/security.html

```diff
@@ -416,7 +448,10 @@
                 the user for client connections. In this example, clients connect to the broker as user <i>alice</i>.
             </li>
             <li>Pass the JAAS config file location as JVM parameter to each client JVM:
-                <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre></li>
+                <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre>
```

Comment:
Thanks! Looks good by looking at the diff. Will check how it looks rendered and hopefully merge soon.